lib/benchpark/experiment.py

```diff
@@ -71,16 +71,17 @@ def compute_config_variables_wrapper(self):
         return self._expr_vars, self.env_vars
 
 
-class SingleNode:
+class ExecMode:
     variant(
-        "single_node",
-        default=True,
-        description="Single node execution mode",
```

Comment:
This change will be blocked until benchmarks using old scaling are at least updated removing check for `single_node`